docs/device-and-supply-chain-modelling.md

```diff
@@ -0,0 +1,78 @@
+# Device and Supply Chain modelling
+
+## Intro
```

Comment:
Introduction would be better here than Intro.

Reply:
Fixed